sql/src/test/java/org/apache/druid/sql/guice/SqlModuleTest.java

```diff
@@ -0,0 +1,256 @@
+import org.apache.druid.query.lookup.LookupExtractorFactoryContainerProvider;
+import org.apache.druid.segment.join.JoinableFactory;
+import org.apache.druid.segment.loading.SegmentLoader;
+import org.apache.druid.server.log.NoopRequestLogger;
+import org.apache.druid.server.log.RequestLogger;
+import org.apache.druid.server.security.AuthorizerMapper;
+import org.apache.druid.server.security.Escalator;
+import org.apache.druid.server.security.NoopEscalator;
+import org.apache.druid.sql.calcite.planner.PlannerFactory;
+import org.apache.druid.sql.calcite.util.CalciteTests;
+import org.apache.druid.sql.calcite.view.DruidViewMacro;
+import org.apache.druid.sql.calcite.view.NoopViewManager;
+import org.apache.druid.sql.calcite.view.ViewManager;
+import org.easymock.EasyMock;
+import org.easymock.EasyMockRunner;
+import org.easymock.Mock;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import javax.validation.Validation;
+import javax.validation.Validator;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+
+@RunWith(EasyMockRunner.class)
+public class SqlModuleTest
+{
+  @Mock
+  private ServiceEmitter serviceEmitter;
+
+  @Mock
+  private InventoryView inventoryView;
+
+  @Mock
+  private TimelineServerView timelineServerView;
+
+  @Mock
+  private DruidLeaderClient druidLeaderClient;
+
+  @Mock
+  private DruidNodeDiscoveryProvider druidNodeDiscoveryProvider;
+
+  @Mock
+  private GenericQueryMetricsFactory genericQueryMetricsFactory;
+
+  @Mock
+  private QuerySegmentWalker querySegmentWalker;
+
+  @Mock
+  private QueryToolChestWarehouse queryToolChestWarehouse;
+
+  @Mock
+  private LookupExtractorFactoryContainerProvider lookupExtractorFactoryContainerProvider;
+
+  @Mock
+  private JoinableFactory joinableFactory;
+
+  @Mock
+  private SegmentLoader segmentLoader;
+
+  private Injector injector;
+
+  @Before
+  public void setUp()
+  {
+    EasyMock.replay(
+        serviceEmitter,
+        inventoryView,
+        timelineServerView,
+        druidLeaderClient,
+        druidNodeDiscoveryProvider,
+        genericQueryMetricsFactory,
+        querySegmentWalker,
+        queryToolChestWarehouse,
+        lookupExtractorFactoryContainerProvider,
+        joinableFactory,
+        segmentLoader
+    );
+  }
+
+  @Test
+  public void testDefaultViewManagerBind()
+  {
+    final Properties props = new Properties();
+    props.setProperty(SqlModule.PROPERTY_SQL_ENABLE, "true");
+    props.setProperty(SqlModule.PROPERTY_SQL_ENABLE_AVATICA, "true");
+    props.setProperty(SqlModule.PROPERTY_SQL_ENABLE_JSON_OVER_HTTP, "true");
+
+    injector = makeInjectorWithProperties(props);
+
+    ViewManager viewManager = injector.getInstance(Key.get(ViewManager.class));
+    Assert.assertNotNull(viewManager);
+    Assert.assertTrue(viewManager instanceof NoopViewManager);
+  }
+
+
+  public void testNonDefaultViewManagerBind()
```

Comment:
missing `@Test` is causing CI failure I think, https://travis-ci.com/github/apache/druid/jobs/483641257#L2095